clickhouse/migrations/schema_46_tags.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE IF NOT EXISTS default.tags (
```

Comment:
I think we should add organization id here as the highest order on the cardinality to help the queries be faster. Also to check auth faster in the SessionManager